### ISSUE TITLE
triangular appearance to lunate bone

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10733]
**Preferred term label:**
triangular appearance to lunate bone

**Synonyms**
"piece-of-pie" sign

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0004248

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Perilunate dislocation (PLD)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001151
- label: Triangular lunate bone
- parents: [Abnormality of the lunate bone (HP:0004248)](https://hpo.jax.org/app/browse/term/HP:0004248)
- pmid: PMID:23616330|PMID:22529764
- def: A disruption of the greater and lesser arcs of the lunate bone, with the lunate having a triangular silhouette, know as the piece of pie sign. Often caused by falling on an outstretched hand, patients will have tenderness to palpation over the dorsum of the wrist and may have sensory deficits in the median nerve distribution.
- comment: See Figure 3 of PMID:23616330.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10733
- synonyms: Piece of pie sign of lunate bone(NONE)
